dev-python/funcsigs/funcsigs-1.0.2.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="Python signature tools"
+DESCRIPTION="Python function signatures from PEP362 for Python 2.6, 2.7 and 3.2+."
+HOMEPAGE="http://funcsigs.readthedocs.org"
+SOURCE_URI="https://pypi.python.org/packages/94/4a/db842e7a0545de1cdb0439bb80e6e42dfe82aaeaadd4072f2263a4fbed23/funcsigs-1.0.2.tar.gz"
+REVISION="1"
+LICENSE="Apache v2"
+COPYRIGHT="Testing Cabal"
```

Comment:
COPYRIGHT should have year or year range listed.  Perhaps this should be "2013 Aaron Iles" ?
https://github.com/testing-cabal/funcsigs/blob/master/LICENSE